website/docs/docs/collaborate/manage-access/audit-log.md

```diff
@@ -5,11 +5,11 @@ description: "You can troubleshoot possible issues and provide security audits b
 sidebar_label: "Audit log"
 ---
 
-To review actions performed by people in your organization, dbt provides logs of audited user and system events. You can use the audit log to quickly review the actions performed by members of your organization. The audit log includes details such as who performed the action, what the action was, and when it was performed. You can use these details to troubleshoot access issues, perform security audits, or analyze specific events. 
+To review actions performed by people in your organization, dbt provides logs of audited user and system events in real-time. You can use the audit log to quickly review the actions performed by members of your organization. The audit log appears as events happen and includes details such as who performed the action, what the action was, and when it was performed. You can use these details to troubleshoot access issues, perform security audits, or analyze specific events. 
```

Comment:
The second sentence restates the first sentence.

```suggestion
To review actions performed by people in your organization, dbt provides logs of audited user and system events in real time. The audit log appears as events happen and includes details such as who performed the action, what the action was, and when it was performed. You can use these details to troubleshoot access issues, perform security audits, or analyze specific events. 
```

Reply:
d'oh! thanks @matthewshaver and great catch!